Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -969,6 +970,13 @@ def update_ticket_command(client: Client, args: dict) -> Tuple[Any, Dict, Dict,
     hr_ = get_ticket_human_readable(ticket, ticket_type, additional_fields_keys)
     human_readable = tableToMarkdown(f'ServiceNow ticket updated successfully\nTicket type: {ticket_type}',
                                      t=hr_, removeNull=True)
+
+    # make the modified fields the user inserted as arguments show in the context
+    if additional_fields:
+        additional_fields_keys.extend(list(args.keys()))
+    else:
+        additional_fields_keys = list(args.keys())
```

Comment:
can there be duplicate keys?

Reply:
not. but would still: list(set(a).union(set(b))
why leave this edge open?
also a good 2nd test case

Reply:
done